test/jdk/java/foreign/TestSegments.java

```diff
@@ -191,6 +190,38 @@ public void testSmallSegmentMin() {
         memorySegment.get(JAVA_INT, offset);
     }
 
+    @Test(expectedExceptions = IndexOutOfBoundsException.class)
+    public void testNegativeGetOffset() {
```

Comment:
We have a test specifically for instance segment accessors (TestMemoryAccessInstance) which covers _all_ accessors - maybe better to move these new tests there?